keps/349-all-or-nothing/README.md

```diff
@@ -76,6 +78,10 @@ large jobs may deadlock if there are issues with resource provisioning to
 match the configured cluster quota. The same pair of jobs could run to
 completion if their pods were scheduled sequentially.
 
+Another reason a pods could not schedule, even if Kueue had a free quota would be resource
+fragmentation. We would like to prevent allocating quota for an extended period of time
+if it cannot be consumed.
```

Comment:
Maybe we can rephrase it a bit to emphasize the admission checks. The connection between resource fragmentation and admission checks is not clear. I think we can mention: 

```
Another reason pods could not schedule is that there are some AdmissionChecks configured which are
not satisfied for a long time for a Workload which already reserves the quota. 

For example, some users may use AdmissionCheck ProvisioningRequest with check-capacity to gate
admission of Workload. If the quota is fragmented the Workload may stay waiting for admission for
an extended period of time, and we would like to help users to avoid that situation.
```

Reply:
Done